react-backend/routes/signup.js

```diff
@@ -0,0 +1,46 @@
+var express = require("express");
+var router = express.Router();
+var mysql = require("mysql");
+var config = require("../config");
+var bodyParser = require("body-parser");
+
+const connection = mysql.createPool({
+  host: config.host,
+  user: config.user,
+  password: config.password,
+  database: config.database
+});
+
+router.post("/", function(req, res, next) {
+  var accountID = req.query.accountID;
+  var email = req.query.email;
+  var password = req.query.password;
+  var balance = req.query.balance;
+  
+  query =
+    "insert into account (accountID, email, password, balance) " + 
+    "values (" + connection.escape(accountID) + ", " + connection.escape(email) + 
+    ", AES_ENCRYPT(" + connection.escape(password) + ", '" + config.password +
+     "', " + connection.escape(balance) + ");";
+
+  testQuery = 
+  "select * from account where accountID = " + connection.escape(accountID) + ";"
```

Comment:
Not sure we need this. If it fails to insert, it will send an error message back